packages/xod-cli/src/args.js

```diff
@@ -0,0 +1,61 @@
+import { cwd } from 'process';
+import path from 'path';
+import fs from 'fs-extra';
+import { compose, when, startsWith, complement, isEmpty } from 'ramda';
+import { getPathToXodProject, resolvePath, isBasename } from 'xod-fs';
+import { isValidPatchPath } from 'xod-project';
+import * as msg from './messages';
+
+// fake entrypoint argument just for --help
+// use with command's property strict == false
+const entrypoint = {
+  name: 'entrypoint',
+  required: false,
+  hidden: false,
+  description: msg.entrypointArgumentDescription,
+};
+
+// convert (projectPath, patchPath) to patch name
+const getPatchName = (projectPath, patchPath) => {
+  const relPath = path.relative(projectPath, patchPath);
+  if (!relPath) return null;
+  return compose(
+    when(complement(isEmpty), pPath => `@/${pPath}`),
+    when(isBasename('patch.xodp'), path.dirname)
+  )(path.relative(projectPath, patchPath));
```

Comment:
Looks like you decided just to turn off `hm-def` 😛

Reply:
No, I filter out `null`, so no error here.

Reply:
🤔 
I'm telling about `isBasename :: String -> Path -> Boolean`, where `Path is not empty string`, but `path.relative` could return empty string if paths are equal...